src/garage/replay_buffer/path_buffer.py

```diff
@@ -9,8 +9,10 @@ class PathBuffer:
 
     This buffer only stores valid steps, and doesn't require paths to
     have a maximum length.
+
     Args:
-        capacity_in_steps (int): total memory allocated for the buffer
+        capacity_in_transitions (int): Total memory allocated for the buffer.
```

Comment:
This is a really long arg name. I suggest we stick with steps since it is also used here https://github.com/rlworkgroup/garage/blob/master/src/garage/_dtypes.py#L410

Reply:
What about `_transitions_stored` and `sample_transitions`? Are you suggesting to also change these to steps?

Reply:
So Linda's just making the docstring consistent with the code I wrote in this case.
This argument is long. Probably the whole `PathBuffer` should use the term step(s), instead of transition(s), but I think that's out of scope for this PR.